src/TreeView/index.tsx

```diff
@@ -419,19 +422,47 @@ const useTree = ({
       nodeRefs?.current != null &&
       leafRefs?.current != null
     ) {
-      const isTreeActive = (treeRef?.current == null) || 
-        (document.activeElement && treeRef.current.contains(document.activeElement));
-      if (isTreeActive) {
+      const isTreeActive =
+        treeRef?.current == null ||
+        (document.activeElement &&
+          treeRef.current.contains(document.activeElement));
+      if (isTreeActive || focusedId) {
         // Only scroll and focus on the tree when it is the active element on the page.
         // This prevents controlled updates from scrolling to the tree and giving it focus.
         const tabbableNode = nodeRefs.current[tabbableId];
-        const leafNode = leafRefs.current[lastInteractedWith]; 
+        const leafNode = leafRefs.current[lastInteractedWith];
         scrollToRef(leafNode);
         focusRef(tabbableNode);
       }
     }
   }, [tabbableId, nodeRefs, leafRefs, lastInteractedWith]);
 
+  //Controlled focus
+  useEffect(() => {
+    if (!focusedId) {
+      dispatch({
+        type: treeTypes.clearFocus,
+        id: treeParentNode.children[0],
```

Comment:
Clear focus. This result in the default uncontrolled tree and focus on first element when they tab on tree.